### ISSUE TITLE
skaffold-autoconfigure-repo: poorly cropped repo

### ISSUE THREAD

**Issue by palladius** [palladius/sakura#7]
current invokation:

```
🐼  skaffold-autoconfigure-repo 
👋 Welcome to skaffold-autoconfigure-repo v1.1
Your active configuration is: [cicd-platinum-32]
[ricc] I presume youre on a mac..
[ricc] I presume youre on a mac..
🤔 This command will help you set up skaffold repo for your k🎱s config: gke_cicd-platinum-test032_europe-west1_cicd-dev
Listing items under project cicd-platinum-test032, across all locations.

Try one of these:
🌻 REPO cicd-plat02 🌍 europe-west1
  🌻 skaffold config set default-repo europe-west1-docker.pkg.dev/cicd-platinum-test032/cicd-plat02
  🌻 gcloud auth configure-docker europe-west1-docker.pkg.dev # This can help with Auth
🌻 REPO pasta-and-friends32 🌍 a
  🌻 skaffold config set default-repo a-docker.pkg.dev/cicd-platinum-test032/pasta-and-friends32
  🌻 gcloud auth configure-docker a-docker.pkg.dev # This can help with Auth
NOTE. If these three are NOT the same, this might give you problems. Particularly if 2<>3 you might need to enable proper SvcAccts to access cross-project which doesnt come by default
1. GCLOUD_PROJECT:      cicd-platinum-test032
2. K8S_PROJECT:         cicd-platinum-test032
3. AR_PROJECT (if set): 
```

unfortunately the second repo should NOT be `a-docker.pkg.dev/cicd-platinum-test032/pasta-and-friends32`, but
`europe-west3-docker.pkg.dev/cicd-platinum-test032/pasta-and-friends32`

so the region is mis-parsed.

**Comment by palladius** [comment]
i got the problem:  gcloud artifacts repositories list  is brittle. At home it has DESCRIPTION before LOCATION. 
So the solution is to force the format as i want it :)

* force DOCKER in search
* just output the two fields I care about: REGION and REPO.

🐼  gcloud artifacts repositories list 2>/dev/null 
                                                                            ARTIFACT_REGISTRY
REPOSITORY           FORMAT  MODE                 DESCRIPTION        LOCATION      LABELS         ENCRYPTION          CREATE_TIME          UPDATE_TIME          SIZE (MB)
cicd-plat02          DOCKER  STANDARD_REPOSITORY                     europe-west1                 Google-managed key  2022-11-03T07:44:04  2022-11-11T17:19:27  1042.586
pasta-and-friends32  DOCKER  STANDARD_REPOSITORY  a cacchio di cane  europe-west3  product=pasta  Google-managed key  2022-11-26T07:35:50  2022-11-26T07:35:50  0

**Comment by palladius** [comment]
I think the magical crafted string is 

```bash
 gcloud artifacts repositories list --format 'table(name.basename(),name.segment(3),sizeBytes.size(zero='0',precision=3,units_out=M))' --filter format='DOCKER'
```

**Comment by palladius** [comment]
```bash
$ gcloud artifacts repositories list --format 'table(name.basename(),name.segment(3),sizeBytes.size(zero='0',precision=3,units_out=M))' --filter format='DOCKER'
Listing items under project cicd-platinum-test032, across all locations.

REPOSITORY           REPOSITORY    SIZE (MB)
cicd-plat02          europe-west1  1042.586
pasta-and-friends32  europe-west3  0
```

(third is not needed but too cute not to have it)

**Comment by palladius** [comment]
fixed in #8947f33e8617187033c1d840b8d76293f697e5ca

**Comment by palladius** [comment]
fixed in #8947f33e8617187033c1d840b8d76293f697e5ca
fixed in # 8947f33e8617187033c1d840b8d76293f697e5ca
